packages/components/src/Autocomplete/Autocomplete.tsx

```diff
@@ -1,5 +1,4 @@
 import React, { useEffect, useRef, useState } from "react";
-// eslint-disable-next-line import/no-internal-modules
```

Comment:
I don't have to clean these up as part of this as it is technically un-related to the core work. But, I would like to.